spec/travis/brakeman.rb

```diff
@@ -0,0 +1,12 @@
+require 'brakeman'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.